config/west.yml

```diff
@@ -7,7 +7,7 @@ manifest:
   projects:
     - name: zmk
       remote: zmkfirmware
-      revision: ae8299edb3d638f1332475b1da0fdf40afa43fe4
+      revision: main
```

Comment:
Can you set the version in the config to match the current 'main' so we don't run into build issues in the future and its clear exactly which version of zmk is being used for ardux builds?